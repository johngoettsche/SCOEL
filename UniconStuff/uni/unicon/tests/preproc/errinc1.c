/* test error massage: no file on #include */

#include
