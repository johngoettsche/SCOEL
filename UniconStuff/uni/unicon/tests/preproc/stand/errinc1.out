pp: File errinc1.c; Line 3: file name missing from #include
